My feature request works as expected
Currently, CI is broken but I think, is not part of my change.
Indeed travis build `likwid` from master so new and old symbol could be missing

```
Traceback (most recent call last):
  File "./tests//testpin.py", line 3, in <module>
    import pylikwid
ImportError: /home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/pylikwid.so: undefined symbol: power_names

```

@TomTheBear could you review it, please?

--- a/setup.py
+++ b/setup.py
@@ -50,7 +50,7 @@ def get_hierarchy():
         library = find_file(iterator)
         if library is not None:
             library_path = os.path.dirname(library)
-    else:
+    if library is None:
         paths_iterator = iter(os.environ["PATH"].split(":"))
         is_searching = True
         while is_searching:

